os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -33,6 +34,8 @@ BLE_TIZENRT_BOND_REQ *ble_tizenrt_bond_req_info = NULL;
 T_TIZENRT_CLIENT_READ_RESULT *ble_read_results = NULL;
 BLE_TIZENRT_APP_LINK *ble_app_link_table = NULL;
 bool (*ble_tizenrt_client_send_msg)(uint16_t sub_type, void *arg) = NULL;
+void *ble_tizenrt_modi_whitelist_sem = NULL;
```

Comment:
Same as above.
(Modify rather than modi)

Reply:
Modify is better.
I have change it in the latest commit.
Thanks